include/FreeRTOS_IP_Private.h

```diff
@@ -916,21 +922,6 @@
 
     #endif /* ipconfigSUPPORT_SELECT_FUNCTION */
 
-    #if ( ipconfigSUPPORT_SELECT_FUNCTION == 1 ) || ( ipconfigUSE_TCP == 1 ) || ( ipconfigDNS_USE_CALLBACKS == 1 )
-        extern ipDECL_CAST_PTR_FUNC_FOR_TYPE( ListItem_t );
-        extern ipDECL_CAST_CONST_PTR_FUNC_FOR_TYPE( ListItem_t );
-    #endif
-
-    extern ipDECL_CAST_PTR_FUNC_FOR_TYPE( NetworkBufferDescriptor_t );
```

Comment:
Now when I compile Sockets.c, it is missing the above CAST functions. Was that a mistake to (re)move them?